src/Entities/Properties/Relation.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace FiveamCode\LaravelNotionApi\Entities\Properties;
+
+use Illuminate\Support\Collection;
+
+/**
+ * Class Relation
+ * @package FiveamCode\LaravelNotionApi\Entities\Properties
+ */
+class Relation extends Property
+{
+    /**
+     * @param $relationIds
+     * @return Relation
+     */
+    public static function instance(array $relationIds): Relation
+    {
+        $relationProperty = new Relation();
+        $relationProperty->content = new Collection();
+        $relationProperty->rawContent = ['relation' => []];
+
+        foreach($relationIds as $relationId){
+            array_push($relationProperty->rawContent['relation'], ['id' => $relationId]);
```

Comment:
should we use laravels array and string helper functions? just a general question, not particularly for this spot.